DEPLOYMENT_TYPES.md

```diff
@@ -14,9 +17,12 @@ The table below outlines the defaults per `deployment_type`.
 | **Image Streams**                                               | centos                                   | rhel + xpaas                           | rhel                             |
 
 
-**NOTE** `enterprise` deployment type is used for OpenShift Enterprise version
-3.0.x OpenShift Enterprise deployments utilizing version 3.1 and later will
-make use of the new `openshift-enterprise` deployment type.  Additional work to
-migrate between the two will be forthcoming.
+**NOTES**:
+
+- `enterprise` deployment type is used for OpenShift Enterprise version
+3.0.x.
+- `openshift-enterprise` deployment type is used for OpenShift Enterprise (and now OpenShift Container Platform) version 3.1 and later.
 
 
+# REVIEW: notes, what versions use what
+# REVIEW: table, what is the registry_url for openshift-enterprise -- isn't it openshift3/ose-...?
```

Comment:
@sosiouxme @sdodson could you please help me review this?

1. The table says `openshift-enterprise` uses `aos3/aos-${component}:${version}`, I have the impression that is incorrect, as per [roles/openshift_facts/library/openshift_facts.py#L862-L863](https://github.com/openshift/openshift-ansible/blob/3ea629b458c78db194443563aab4a8d09e9a07b4/roles/openshift_facts/library/openshift_facts.py#L862-L863).
2. Are the notes above about `enterprise` x `openshift-enterprise` reasonable? Shall we drop the documentation about `enterprise` altogether?